pos_pricelist_technical/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module extends the functionality of point of sale to prevent
+to select in the point of sale, technical pricelists that used for the
+computation of other pricelists, but that should not be selected.
```

Comment:
```suggestion
This module extends the functionality of Point of Sale to prevent
technical pricelists that are used for the
computation of other pricelists from being selected.
```